pkg/autohealing/cloudprovider/openstack/provider.go

```diff
@@ -250,6 +250,61 @@ func (provider OpenStackCloudProvider) waitForServerDetachVolumes(serverID strin
 	return rootVolumeID, err
 }
 
+// FirstTimeRepair Handle the first time repair for a node
+// 1) If the node is the first time in error, reboot it
```

Comment:
reboot and uncordon it.

Also, please describe explicitly what's the meaning of the returned value(`processed`) and what the called should do.